kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/DefaultExpressionEvaluator.java

```diff
@@ -156,6 +157,19 @@ ExpressionTransformResult visitComparator(Predicate predicate) {
             }
         }
 
+        @Override
```

Comment:
Can this be combined into the `visitComparator` above? That way we can avoid an additional method on this visitor.